content/releases/5/4_batch-delete/snippet.php

```diff
@@ -0,0 +1,33 @@
+<style>
+.v5-batch-columns {
+	display: grid;
+	gap: var(--spacing-6);
+	grid-template-areas:
+		"screenshot"
+		"teaser"
+		"code"
+}
+
+@media screen and (min-width: 80rem) {
+	.v5-batch-columns {
+		grid-template-columns: 1fr 1fr 2fr;
+		grid-template-areas:
+			"teaser code screenshot"
+	}
+}
+</style>
+
+<div class="v5-batch-columns">
+	<figure class="release-box bg-light grid place-items-center" style="grid-area: screenshot">
+		<?php snippet('templates/release-4/image', [
+			'alt' => '',
+			'img' => $section->image('batch-delete.png')->resize(850)
```

Comment:
Needs an actual alt text. And also not sure about the size here either.

Reply:
I've added the alt text. Size looks good.